fv_be/app/models/dictionary.py

```diff
@@ -2,39 +2,50 @@
 from django.utils.translation import gettext as _
 
 # FirstVoices
+from .base import BaseModel
 from .sites import BaseSiteContentModel, BaseControlledSiteContentModel
 from .category import Category
 from .part_of_speech import PartOfSpeech
 
 
-class DictionaryNote(BaseSiteContentModel):
+class BaseDictionaryContentModel(BaseModel):
+    """
+    Base model for Dictionary models which all have DictionaryEntry as a foreign key and
+    require site as a property but not as a field.
```

Comment:
Minor comment edit: these "require" a DictionaryEntry and "have" site as a property.